deltacat/__init__.py

```diff
@@ -44,7 +44,7 @@
 
 deltacat.logs.configure_deltacat_logger(logging.getLogger(__name__))
 
-__version__ = "1.1.7"
+__version__ = "1.1.8"
```

Comment:
Let's not bump yet. I have some other changes to go in today.

Reply:
Sg, let's coordinate on the version bump.

Reply:
Reverted version bump.